host.go

```diff
@@ -217,9 +206,9 @@ func (h *Host) StartDocker() error {
 
 	switch h.Driver.GetProviderType() {
 	case provider.Local:
-		cmd, err = h.GetSSHCommand("sudo /etc/init.d/docker start")
+		_, _, err = h.GetSSHCommand("sudo /etc/init.d/docker start")
```

Comment:
These cannot be a blank identifier as the `cmd.Run()` will be nil.  https://gist.github.com/ehazlett/bbc204d5111d82b3e6b7